apps/video/js/db.js

```diff
@@ -1,7 +1,13 @@
+/* global videodb:true,storageState:true,firstScanEnded:true */
+/* global MediaDB,updateDialog,playerShowing,hidePlayer,updateLoadingSpinner */
```

Comment:
This is a lot of globals. One option is to either split them into multiple lines, or all one comment. E.g.,

```
/* global videodb:true,storageState:true,firstScanEnded:true
  MediaDB,updateDialog,playerShowing,hidePlayer,updateLoadingSpinner
*/
```